aquarius/app/es_instance.py

```diff
@@ -105,7 +105,7 @@ def read(self, resource_id):
         :param resource_id: id of the object to be read.
         :return: object value from elasticsearch.
         """
-        logger.debug("elasticsearch::read::{}".format(resource_id))
+        # logger.debug("elasticsearch::read::{}".format(resource_id))
```

Comment:
Should we add a log here as well?

Reply:
no, this will flood the console with reads for non-existing assets because of non-ocean NFT transfers